Sources/ClientRuntime/Networking/Http/SdkHttpRequest.swift

```diff
@@ -155,9 +161,11 @@ extension SdkHttpRequest: CustomDebugStringConvertible, CustomStringConvertible
 
     public var description: String {
         let method = method.rawValue.uppercased()
-        let protocolType = endpoint.protocolType ?? ProtocolType.https
-        let query = String(describing: queryItems)
-        return "\(method) \(protocolType):\(endpoint.port) \n Path: \(endpoint.path) \n \(headers) \n \(query)"
+        let protocolType = self.destination.scheme
+        let query = self.destination.queryString ?? ""
+        let port = self.destination.port
+        return "\(method) \(protocolType):\(port) \n " +
```

Comment:
Handle swift warning that gets raised here. One way to do so is by providing default value after nil-coalescing operator like this: `:\(port ?? "no-port-found")`